app/views/groups/questions/written/answered/index.html.haml

```diff
@@ -0,0 +1,16 @@
+.section--primary
+  .container
+    %h1
+      %span= @answering_body.name
+
+%section{ id: 'content', tabindex: '0' }
+  .container
+    - @answers_grouped_by_date.each do |date, answers|
+      %h2
+        %time{ datetime: l(date, format: '%Y-%m-%d') }= l(date, format: '%d %B %Y') if date
```

Comment:
If there is no date, this will render an empty `h2` @UsmanAfzal is this okay?

Reply:
@mattrayner definitely not okay. Thanks for pointing out.